experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -85,30 +86,38 @@ if mode_name == "amip"
     println("No ocean sim - do not expect energy conservation")
 
     # ocean
-    weightfile, datafile_cgll, regrid_space =
-        ncreader_rll_to_cgll_from_space(sst_data, "SST", boundary_space, outfile = "sst_cgll.nc")
-    SST_init = ncreader_cgll_sparse_to_field(datafile_cgll, "SST", weightfile, (Int(1),), regrid_space)[1]
-    SST_init = swap_space!(SST_init, axes(mask)) .* (abs.(mask .- 1)) .+ FT(273.15)
-
+    SST_info = bcfile_info_init(
+        sst_data,
+        "SST",
+        boundary_space,
+        segment_idx0 = [Int(1309)],
```

Comment:
hardcoded 1309, is this ok?

Reply:
I added a function that finds the closest index of the file date to the user-elected start date.